H2O/introduction-to-machine-learning-with-h2o-3-regression/introduction-to-machine-learning-with-h2o-3-regression.md

```diff
@@ -0,0 +1,819 @@
+
+### Deep Learning 
+Deep learning is a subset of machine learning that uses neural networks to solve complex problems that other machine learning algorithms have difficulties with, such as image classification, speech recognition, etc. Deep Learning models have an advantage over standard machine learning because they are very effective at learning non-linear derived features from the input features. Neural networks are algorithms based on how the human brain works because they are composed of nodes or commonly known as “neurons,” and thus the name of neural networks. One of the simplest neural network architecture is a single layer perceptron. A perceptron takes binary inputs, as shown in the picture below, and produces a single binary output[2]. 
+
+![perceptron](assets/perceptron.jpg)
+
+In the figure above, we see a perceptron with three inputs; these inputs are fed into the neuron, which will then decide if the output should be a zero or one. Simply said, a neuron is a function that takes multiple numeric inputs and yields a numeric output. Below is a representation of a more complex neural network
+
+![deep-neural-network](assets/deep-neural-network.jpg)
+
+In the image above, you can see a neural network with two hidden layers, and with an input and output layer. We can say that the input layer has six neurons, the first hidden layer has four neurons, while the second hidden layer has 3, and lastly, the output layer has one neuron. A neural network can have as many hidden layers as the user may find suitable. 
+Although deep learning models can solve almost any problem, they tend to be slow and can be hard to use and tune. Nowadays, none of the most popular deep learning frameworks is TensorFlow, as it is an open-source AI library that allows users to create different types of neural networks with many layers. Tensorflow allows you to build Deep Neural Networks(DNNs), which are feedforward multilayer neural networks, among other types of models. TensorFlow also provides DNNs models for classification and for regression use cases as different models. 

```

Comment:
@FrankJVA this sentence is a bit confusing consider written it: 

Nowadays, none of the most popular deep learning frameworks is TensorFlow